configure.sh

```diff
@@ -60,17 +73,8 @@ config_apps() {
 	ooc config:app:set --value="no" core shareapi_only_share_with_group_members
 	ooc config:app:set --value='["admin"]' core shareapi_only_share_with_group_members_exclude_group_list
 
-	echo "Configure nc_ionos_processes app"
-
-	if [ -z "${IONOS_PROCESSES_API_URL}" ] || [ -z "${IONOS_PROCESSES_USER}" ] || [ -z "${IONOS_PROCESSES_PASS}" ]; then
-		echo "Warning: IONOS_PROCESSES_API_URL, IONOS_PROCESSES_USER or IONOS_PROCESSES_PASS not set, skipping configuration of nc_ionos_processes app"
-		return
-	fi
+	configure_app_nc_ionos_processes
```

Comment:
I moved the block in own function.. ..so the script is not aborted

Reply:
I will just add some color :))